mygame.py

```diff
@@ -98,7 +99,16 @@ def Step(self):
                     print("You cannot get that.")
             elif lowcmd in self.inventory_commands:
                 for item in self.player1.display_inv():
-                    self.graphics.RenderText(item)              
+                    self.graphics.RenderText(item)
+            elif lowcmd in self.use_commands:
+                item_use = input("What will you use?")
+                if item_use.lower() == 'key':
+                    if 'a key' in self.player1.display_inv():
```

Comment:
I think this is great for now, its a simple approach to adding this feature, but if we want to expand on this we should find a more systematic way to evaluate inventory items and not use their description strings as their id